fala/templates/generic-py-base.html

```diff
@@ -14,36 +15,99 @@
   <body class="govuk-template__body {{ 'fala-' + ENVIRONMENT }} govuk-frontend-supported">
     <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
     <header class="govuk-header " role="banner" data-module="govuk-header">
-      <div id="cookie-message" class="laa-cookie-banner
-        {% if request.COOKIES.get('cookiePermission') == 'Allowed' %}
-          laa-cookies-accepted
-        {% elif request.COOKIES.get('cookiePermission') == 'Rejected' %}
-          laa-cookies-rejected
-        {% endif %}
-        " data-module="cookie-banner" role="region" aria-label="cookie banner">
-        <div class="govuk-width-container govuk-!-padding-top-3">
-          <div class="govuk-grid-row">
-            <div class=" govuk-grid-column-three-quarters">
-              <div>
-                <span class="govuk-heading-m">{% trans %}Cookies on Find a legal adviser or family mediator{% endtrans %}</span>
-                <p class="govuk-body">
-                  {% set cookie_info_url = '/cookies' %}
-                  <p>{% trans %}We use some essential cookies to make this website work.{% endtrans %}</p>
-                  <p>{% trans %}We’d like to set additional cookies so we can remember your settings, understand how people use the service and make improvements.{% endtrans %}</p>
-                </p>
-              </div>
-              <div class="govuk-button-group">
-                <button id="cookieAccept" class="govuk-button" type="submit" data-module="track-click" data-accept-cookies="true" data-track-category="cookieBanner" data-track-action="Cookie banner accepted">
-                  {% trans %}Accept analytics cookies{% endtrans %}
-                </button>
-                <button id="cookieReject" class="govuk-button govuk-button--secondary" data-module="track-click" data-accept-cookies="false" data-track-category="cookieBanner" data-track-action="Cookie banner rejected">
-                  {% trans %}Reject analytics cookies{% endtrans %}
-                </button>
-                {% trans %}<a id="cookies_banner_view_link" class="govuk-link" href="/cookies">View cookies</a>{% endtrans %}
-              </div>
-            </div>
-          </div>
-        </div>
+      <div class="fala-cookie-banner">
```

Comment:
Because this is so big (Which is correct) I think this is worth placing into it's own HTML file and then being included.

When naming a sub html file to be included prefix it with an underscore `_`, So for example `_CookieBanner.html`